### PR TITLE
adds standard turfmakers

### PR DESCRIPTION
**Pull request by silicons**
/indoors - tin
/outdoors - tin
/overhang - not outdoors, but outdoors atmos
/default - use area defaults

default is currently inefficient (5  bytes per var :/) but whatever

as of right now everything uses zlevel default atmos for outdoors, and indoors uses STP
we'll have better stuff on level side later

### CODE REVIEW

**Review by TheLordME — APPROVED** [review]

File: code/__DEFINES/turfs/type_generation.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#define CREATE_STANDARD_TURFS(type) \
+##type/indoors; \
+##type/indoors/outdoors = FALSE; \
+##type/indoors/initial_gas_mix = ATMOSPHERE_USE_INDOORS; \
+##type/outdoors/outdoors = TRUE; \
+##type/outdoors/initial_gas_mix = ATMOSPHERE_USE_OUTDOORS; \
+##type/default/outdoors = null; \
+##type/default/initial_gas_mix = ATMOSPHERE_USE_AREA; \
+##type/overhang/outdoors = FALSE; \
+##type/overhand/initial_gas_mix = ATMOSPHERE_USE_OUTDOORS;
```
Comment (Zandario) [inline code/__DEFINES/turfs/type_generation.dm:11]: I'd really like to use the names `internal`/`external` but we can probably do that in a different PR

File: code/game/area/area.dm [hunk 2]
```diff
@@ -4,20 +4,38 @@
  * A grouping of tiles into a logical space, mostly used by map editors
  */
 /area
+	level = null
+	name = "Unknown"
+	icon = 'icons/turf/areas.dmi'
+	icon_state = "unknown"
+	plane = ABOVE_LIGHTING_PLANE //In case we color them
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```
Comment (Zandario) [inline code/game/area/area.dm:12]: No abstract_type? 🥺

**Review by Zandario — APPROVED** [review]
